xc7/make/project_xray.cmake

```diff
@@ -212,15 +214,33 @@ function(PROJECT_XRAY_PREPARE_DATABASE)
   file(GLOB DEPS ${PRJXRAY_DB_DIR}/${PART}/*.json)
   file(GLOB DEPS2 ${PRJXRAY_DIR}/prjxray/*.py)
 
+  file(MAKE_DIRECTORY ${PRJXRAY_DB_OVERLAY_DIR}/${PART})
+
+  set(GRID_SPLITTER ${symbiflow-arch-defs_SOURCE_DIR}/xc7/utils/splitter/grid_splitter.py)
+
+  # FIXME: Move it somewhere
+  set(TILES_TO_SPLIT CLBLL_L CLBLL_R CLBLM_L CLBLM_R)
+
+  set(TILEGRID ${PRJXRAY_DB_OVERLAY_DIR}/${PART}/tilegrid.json)
+  set(TILECONN ${PRJXRAY_DB_OVERLAY_DIR}/${PART}/tileconn.json)
+  add_custom_command(
+  OUTPUT ${TILEGRID} ${TILECONN} # FIXME: The script will output not only these files
```

Comment:
If the overlay is config driven, the output list can be generated from the config?

Reply:
True.